common/analysis/lint_rule_status.cc

```diff
@@ -76,12 +79,14 @@ static TokenInfo SymbolToToken(const Symbol& root) {
 
 LintViolation::LintViolation(const Symbol& root, absl::string_view reason,
                              const SyntaxTreeContext& context,
-                             const std::vector<AutoFix>& autofixes)
+                             const std::vector<AutoFix>& autofixes,
+                             const std::vector<TokenInfo>& tokens)
```

Comment:
nit: name this `related_tokens` for consistency

Reply:
done.